sda-commons-server-prometheus/src/test/java/org/sdase/commons/server/prometheus/PrometheusBundleTest.java

```diff
@@ -240,6 +242,46 @@ void micrometerMetricsAvailableInPrometheus2() {
     assertThat(sampleList.get(0).value).isEqualTo(2);
   }
 
+  @Test
+  void micrometerJVMMetricsAvailable() {
+
+    MeterRegistry globalRegistry = Metrics.globalRegistry;
+
+    List<Meter> meters = globalRegistry.getMeters();
+
+    List<Meter> jvmBufferList =
+        meters.stream()
+            .filter(m -> m.getId().getName().startsWith("jvm.buffer"))
+            .collect(Collectors.toList());
+    List<Meter> jvmMemoryList =
+        meters.stream()
+            .filter(m -> m.getId().getName().startsWith("jvm.memory"))
+            .collect(Collectors.toList());
+    List<Meter> jvmProcessorList =
+        meters.stream()
+            .filter(m -> m.getId().getName().contains("cpu"))
+            .collect(Collectors.toList());
+    List<Meter> jvmThreadsList =
+        meters.stream()
+            .filter(m -> m.getId().getName().startsWith("jvm.threads"))
+            .collect(Collectors.toList());
+    List<Meter> jvmClassesList =
+        meters.stream()
+            .filter(m -> m.getId().getName().startsWith("jvm.classes"))
+            .collect(Collectors.toList());
+    List<Meter> jvmGcList =
+        meters.stream()
+            .filter(m -> m.getId().getName().startsWith("jvm.gc"))
+            .collect(Collectors.toList());
+
+    assertThat(jvmBufferList).hasSize(6);
+    assertThat(jvmMemoryList).hasSize(24);
+    assertThat(jvmProcessorList).hasSize(3);
+    assertThat(jvmThreadsList).hasSize(10);
+    assertThat(jvmClassesList).hasSize(2);
+    assertThat(jvmGcList).hasSize(5);
```

Comment:
Tests are failing now for Java 17. Maybe too strict / number of meters depend on the Java version?

Reply:
Yes, metrics depend on Java version :worried:
I've made the tests less strict